var/spack/repos/builtin/packages/py-morphio/package.py

```diff
@@ -10,9 +10,11 @@ class PyMorphio(PythonPackage):
     """Python library for reading / writing morphology files"""
 
     homepage = "https://github.com/BlueBrain/MorphIO/"
+    url      = "https://github.com/BlueBrain/MorphIO.git"
     git      = "https://github.com/BlueBrain/MorphIO.git"
 
-    version('develop', branch='master', submodules=True, clean=False)
+    version('develop', git=url, submodules=True, clean=False)
```

Comment:
Why this change?

Reply:
Since the `morphio` package was working, I took it from there to see if that would solve the issue. In the meantime, I found out the main difference between `morphio` and `py-morphio` is that the later has BUILD_BINDINGS=ON.
TLDR: I could revert that, I guess